explorer/src/components/TwoColSmallTable.tsx

```diff
@@ -29,63 +27,49 @@ export const TwoColSmallTable: React.FC<TableProps> = ({
   values,
   marginBottom,
   error,
-}) => {
-  const theme = useTheme();
-  const { mode } = React.useContext(MainContext);
-  return (
-    <>
-      {title && <Typography sx={{ marginTop: 2 }}>{title}</Typography>}
+}) => (
+  <>
+    {title && <Typography sx={{ marginTop: 2 }}>{title}</Typography>}
 
-      <TableContainer
-        component={Paper}
-        sx={marginBottom ? { marginBottom: 4, marginTop: 2 } : { marginTop: 2 }}
-      >
-        <Table aria-label="two col small table">
-          <TableBody>
-            {keys.map((each: string, i: number) => (
-              <TableRow
-                key={each}
-                sx={{
-                  background:
-                    mode === 'dark'
-                      ? theme.palette.secondary.dark
-                      : theme.palette.primary.light,
-                }}
-              >
-                {icons && (
-                  <TableCell>
-                    {icons[i] ? <CheckCircleSharpIcon /> : <ErrorIcon />}
-                  </TableCell>
-                )}
-                <TableCell sx={error ? { opacity: 0.4 } : null}>
-                  {each}
+    <TableContainer
+      component={Paper}
+      sx={marginBottom ? { marginBottom: 4, marginTop: 2 } : { marginTop: 2 }}
+    >
+      <Table aria-label="two col small table">
+        <TableBody>
+          {keys.map((each: string, i: number) => (
+            <TableRow key={each}>
+              {icons && (
+                <TableCell>
+                  {icons[i] ? <CheckCircleSharpIcon /> : <ErrorIcon />}
```

Comment:
Slightly off topic and maybe could be looked at some other time but this feels a bit difficult to follow. Might be worth a small refactor on this component at some point.

Reply:
Yes, I'm sure there's a better MUI way to do this.